code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -382,8 +382,22 @@
 	name = "gold necklace"
 	desc = "Damn, it feels good to be a gangster."
 	icon_state = "bling"
+	item_state = "bling"
 	item_color = "bling"
-	item_color = "bling"
+
+/obj/item/clothing/accessory/necklace/skullcodpiece
+	name = "skull codpiece"
+	desc = "A skull shaped ornament, intended to protect the important things in life."
+	icon_state = "skull"
+	item_state = "skull"
+	item_color = "skull"
+
+/obj/item/clothing/accessory/necklace/talisman
```

Comment:
isn't this supposed to give a bit of armor?

Reply:
Aye, same with the bone talisman. Forgot about that, easy fix.

Reply:
I can't remember if I added in a way to prevent armor stacking from multiple accessories or not.

I believe so, but I can't remember.

Otherwise, stacking talismans would pretty much mean you could become immune to brute damage.